.circleci/config.yml

```diff
@@ -192,14 +203,22 @@ jobs:
               export PYTHONPATH=$PYTHONPATH:$(pwd)
               GTEST_COLOR=yes ./build.sh --headless --run-tests
               pytest
+      - run:
```

Comment:
This step increases whole testing for 4 min. Unfortunately, when `background: true` option is used step can't fail the testing and is consider successful. Let's keep it as is right now and parallelize later when we have more understanding what other tests we will run for JS build, physics, C++.